installation/standard.py

```diff
@@ -48,8 +48,8 @@
 #     If you set x11libdir (that is two ones and an el) AND x11include to point
 #     to the lib and include directories, they will be used.
 #     Otherwise a search is made for common locations.
-x11libdir=''
-x11include=''
+x11libdir='/usr/X11R6/lib'
+x11include='/usr/X11R6/include'
```

Comment:
yes XQuartz is now a hard requirment for matplotib, @dnadeau4 I don't want it to be hard requirement for libcdms, let's add a check and turn on png for cdms2 only if found (via function I asked for before)